nibabel/streamlines/tractogram.py

```diff
@@ -651,29 +650,46 @@ def apply_affine(self, affine):
         ----------
         affine : 2D array (4,4)
             Transformation matrix that will be applied on each streamline.
+        lazy : True, optional
```

Comment:
Okay, I don't think I can do it without creating a mess in `Tractogram` :-(. I think the best I can do right now is to rename `LazyTractogram` to something like `MemmapTractogram` but that might be a little misleading as it doesn't need a file to create such object (it needs generator functions which can return data stored in memory). If you have some ideas, I'm all ears.